includes/class-wc-siftscience-html.php

```diff
@@ -133,25 +133,29 @@ public function display_batch_table() {
 			<?php
 		}
 		/**
-		 * This function displayes Reporting header receiving the anonymous id from the admin class
+		 * This function displays sub title in repotring page
+		 */
+		public function display_reporting_sub_title() {
+			ob_start();
+			?>
+				Help us improve this plugin by automatically reporting errors and statistics.<br />
+				<em>More info</em> <a target="_blank" href="https://github.com/Fermiac/woocommerce-siftscience/wiki/Statistics-Collection">here</a>.
+			<?php
+			return ob_get_clean();
+		}
+		/**
+		 * This function adds returns anchored reset url
 		 *
-		 * @param String $anonymous_id The id sent from admin genarated by options.
-		 * @param String $reset_url    A url to reset the anonymous id.
+		 * @param String $anonymous_id the $anonymous id revieved from admin genarated by options.
+		 * @param String $url the link in which it should dusokay as a button.
 		 */
-		public function display_reporting_text( $anonymous_id, $reset_url ) {
+		public function display_reset_anchor( $anonymous_id, $url ) {
```

Comment:
Let's rename this and the other function to `get_` instead of `display_`.

`display_` -> echos the result and returns void
`get_` -> returns a string

Reply:
Without reading ok